evasquez26/angular4-startup
# S5-ONE-96 Edit organization

## Story / Task

see [ONE-96](https://bixlabs.atlassian.net/browse/ONE-96)


## What this PR does?

Adds functionality to edit an organization

### Before 

There was no way of editing an organization

### After

Now, we can edit an organization

### How to test / reproduce

- Login to the app with a user with admin roles in an organization
- Go to schools menu
- Select one of the user's schools
- You should see an edit menu. Click edit.
- Update the school info
- Click save
- Check the info was updated.

### TODOs
- [ ] Rebase after merging #40 


## Migrations

NO


Thank you.
